src/WorkflowManager/WorkflowExecuter/Services/WorkflowExecuterService.cs

```diff
@@ -112,13 +112,8 @@ public async Task<bool> ProcessPayload(WorkflowRequestEvent message, Payload pay
             }
             else
             {
-                var aeTitles = new List<string>
-                {
-                    message.CalledAeTitle,
-                    message.CallingAeTitle
-                };
-
-                workflows = await _workflowRepository.GetWorkflowsByAeTitleAsync(aeTitles) as List<WorkflowRevision>;
+                var result = await _workflowRepository.GetWorkflowsForWorkflowRequestAsync(message.CalledAeTitle, message.CallingAeTitle);
```

Comment:
Need to look at further tests to cover this change

Reply:
Added int tests to cover this